### PR TITLE
[4][scheduler] sort by last run

### PR DESCRIPTION
**Pull request by alikon**
### Summary of Changes
add sort by last run date

### Testing Instructions
apply pr
run some tasks

### Actual result BEFORE applying this Pull Request
N/A

### Expected result AFTER applying this Pull Request

image

### CODE REVIEW

**Comment by brianteeman** [comment]
this PR appears to be missing files. Your screen shot shows the sort direction indicator in the table column header.

**Comment by alikon** [comment]
correct, my bad, should be ok now

**Comment by Kostelano** [comment]
I have tested this item :white_check_mark: successfully on 0d393ce179cb722a0d93adb013e18780dbdb4a0a<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37501">issues.joomla.org/tracker/joomla-cms/37501</a>.</sub>

File: administrator/language/en-GB/com_scheduler.ini [hunk 1]
```diff
@@ -77,7 +77,9 @@ COM_SCHEDULER_LABEL_TASK_PRIORITY_LOW="Low"
 COM_SCHEDULER_LABEL_TASK_PRIORITY_NORMAL="Normal"
 COM_SCHEDULER_LABEL_TIMES_EXEC="Times Executed"
 COM_SCHEDULER_LABEL_TIMES_FAIL="Times Failed"
+COM_SCHEDULER_LAST_RUN_ASC="Last Run Ascending"
```
Comment (Kostelano) [inline administrator/language/en-GB/com_scheduler.ini]: May be..? To match all other similar places.

```
COM_SCHEDULER_LAST_RUN_DATE="Last Run Date"
COM_SCHEDULER_LAST_RUN_DATE_ASC="Last Run Ascending"
COM_SCHEDULER_LAST_RUN_DATE_DESC="Last Run Descending"
```

Comment (brianteeman) [reply administrator/language/en-GB/com_scheduler.ini]: also Ascending and Descending should be lowercase for consistency

Comment (alikon) [reply administrator/language/en-GB/com_scheduler.ini]: done

**Comment by heelc29** [comment]
I don't know if it matters, but the heading and the sort label are not the same.

image

**Comment by kiki-G** [comment]
I have tested this item :white_check_mark: successfully on 7918dd42a2c445fdab3b15f1a59ffdeedc39a318<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37501">issues.joomla.org/tracker/joomla-cms/37501</a>.</sub>

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 8f8322ab9f3fd646043655086a2b7f33bc7adbee<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37501">issues.joomla.org/tracker/joomla-cms/37501</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37501">issues.joomla.org/tracker/joomla-cms/37501</a>.</sub>

**Comment by heelc29** [comment]
Can this comment be removed?
https://github.com/joomla/joomla-cms/blob/8f8322ab9f3fd646043655086a2b7f33bc7adbee/administrator/components/com_scheduler/forms/filter_tasks.xml#L43

**Comment by laoneo** [comment]
@heelc29 guess so. Can yo do a pr for this?

**Comment by laoneo** [comment]
Thanks!

**Pull request merged**